app/3.0/start/toolbox/add-elements.md

```diff
@@ -15,25 +15,25 @@ to some off-the-shelf components, for example from [webcomponents.org][webcompon
 ## Install an off-the-shelf component
 
 Once you've identified a component you'd like to install, you'll want to find
-the bower package name for the component.
+the npm package name for the component.
 
 In this step, you'll add Polymer's `<paper-checkbox>` element to your app, which is listed on
-[webcomponents.org][paper-checkbox].  You can use Bower to install it.
+[webcomponents.org][paper-checkbox].  You can use Yarn to install it.
 
 Run this command from your project root directory:
 
-    bower install --save PolymerElements/paper-checkbox
+    yarn add --flat @polymer/paper-checkbox
```

Comment:
Are we recommending yarn? I was never really sure where we landed on yarn vs. npm but I've been using `npm` lately. Also, we may need to explicitly point out that this doesn't work at the moment unless you install `@polymer/paper-checkbox@next` (or some other tag pointing at a 3.0 prerelease version).

Reply:
Also, I just discovered that `polymer serve` seems to cache failed module resolution results. So if the user forgets to install or installs the wrong version and it fails to resolve the first time the specifier is encountered, then they will continue to see that error until they restart the server - even if they get it correctly installed later.

This was a problem for me here since I tried installing `@polymer/paper-checkbox` (without `@next`) which has a year-old release available (v0.0.4) that looks like Polymer 2 code: it has `paper-checkbox.html` but not `paper-checkbox.js`, causing the specifier lookup to fail.

Reply:
(filed as https://github.com/Polymer/tools/issues/238)

Reply:
@keanulee [says that we're recommending `npm` now](https://github.com/Polymer/docs/pull/2539#discussion_r184543074).

Reply:
(Moved discussion to an email thread)